src/contracts/view.html

```diff
@@ -23,7 +23,7 @@ <h1 class="page-header">State of California Contracts</h1>
       The State of California has Master Service Agreements (MSAs) available
       to help you modernize your transit system. Click the contract number below to view all Master Service
       Agreement (MSA) documentation. If you need further assistance as you make your selections,
-      you can <a href="/contact">email us</a>.
+      you can <a href="mailto:hello@calitp.org&subject=Question%20about%20contracts">email us</a>.
```

Comment:
```suggestion
      you can <a href="mailto:hello@calitp.org?subject=Question%20about%20contracts">email us</a>.
```

Should be `?` since it's the first parameter